### DISCUSSION TITLE
Default name setting routine/scheme

### DISCUSSION THREAD

**Discussion by krzywon** [SasView/sasview#2202] [Ideas]
As outlined in PR #1713, a useful feature would be to set a default naming scheme for display names in the data explorer. This scheme could include other metadata, including temperature, detector distance, etc. in the datafile.

**Comment by lucas-wilkins** [comment]
Seems like a good idea, nobody has volunteered yet, and it needs fleshing out. Migrating to a discussion.

**Comment by krzywon** [comment]
Related issue #2083
